docs/Irresponsible-Module.md

```diff
@@ -4,6 +4,10 @@
 
 Classes and modules are the units of reuse and release. It is therefore considered good practice to annotate every class and module with a brief comment outlining its responsibilities.
 
+For further guide on how to write good documentation in Ruby, see this link:
+- <http://rdoc.sourceforge.net/doc/index.html>
+- <http://edgeguides.rubyonrails.org/api_documentation_guidelines.html>
```

Comment:
And let's transform that to "real" links using markdown so that it shows up like this:

* [Rails API documentation guidelines](http://edgeguides.rubyonrails.org/api_documentation_guidelines.html)